src/adapter/z-stack/adapter/zStackAdapter.ts

```diff
@@ -82,7 +82,12 @@ class ZStackAdapter extends Adapter {
     public async start(): Promise<StartResult> {
         await this.znp.open();
 
-        this.version = (await this.znp.request(Subsystem.SYS, 'version', {})).payload;
+        // Old firmware did not contain version information and parse with error now
+        try {
+            this.version = (await this.znp.request(Subsystem.SYS, 'version', {})).payload;
+        } catch (e) {
+            this.version = {"transportrev":2, "product":0, "majorrel":2, "minorrel":0, "maintrel":0, "revision":""};
```

Comment:
What is logged here when adding `console.log(e)`?

Reply:
https://pastebin.com/g8Lg8yAi

Reply:
i.e. ```Error: SRSP - SYS - version after 6000ms```